packages/amplify-provider-awscloudformation/src/push-resources.ts

```diff
@@ -1176,6 +1197,20 @@ export const formNestedStack = async (
         }
         if (resourceDetails.providerMetadata) {
           templateURL = resourceDetails.providerMetadata.s3TemplateURL;
+          const urlParameters = urlParametersMap[templateURL];
```

Comment:
Is there anything that keeps this logic from rolling up into the new code you wrote above starting on line 1059?